kolibri_exercise_perseus_plugin/downloadTranslations.js

```diff
@@ -0,0 +1,65 @@
+const {
+  localeFile,
+  crowdinLangCode,
+  localLocaleFile,
+  remoteLocaleFile,
+  kolibriProjectId,
+  downloadTranslations,
+  languages,
+} = require('./translations');
+const {
+  kolibriApiKey,
+} = require('../crowdinSecrets');
+const fs = require('fs');
+
+const downloadTranslation = (language) => {
+  // Use the language object above to get relevant codes for the language
+
+  const lang = languages[language].lang;
+
+  const locale = languages[language].locale;
+
+  const crowdinLocale = languages[language].crowdinLocale;
+
+  return new Promise((resolve, reject) => {
+    // Download the po file first
+    const pofileDownloadPromise = downloadTranslations(kolibriProjectId, lang, crowdinLocale, kolibriApiKey);
+
+    pofileDownloadPromise.then(zip => {
+
+      console.log(`Finding translations from Kolibri translation files for ${language}`);
+
+      let translationFile = zip.getEntries().find(file => {
+        // Check if this is our message file
+        return file.entryName === remoteLocaleFile(lang, locale)
+      // If so, get the text from the file
```

Comment:
This comment doesn't seem to be in the right place.

Reply:
GOOOONE!